content/en/docs/contribute/generate-ref-docs/metrics-reference.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Generating reference documentation for metrics
+---
+
+<!-- overview -->
+
+This page demonstrates the generation of metrics reference documentation
+
+### Requirements

```

Comment:
```suggestion
## {{% heading "prerequisites" %}}

{{< include "prerequisites-ref-docs.md" >}}
```

Reply:
I can make this change, but we don't really need more than the mentioned things for this generation,
_unless_ we're looking forward to add the generation to the python script in future (which is a good idea imo)